frontend/public/components/dashboards-page/with-dashboard-resources.tsx

```diff
@@ -84,11 +114,27 @@ type WithDashboardResourcesProps = {
   [RESULTS_TYPE.URL]: ImmutableMap<string, any>;
 };
 
+export type FirehoseResource = {
```

Comment:
We should try to move this to a location where this type can be used in other contexts.

Reply:
moved to `module/k8s/index.ts`. Im not sure if there is more appropriate place.

Similar type was used in `factory/details` so I reused it there too.

Reply:
Yeah, I'm not sure the best place either. @alecmerdler ?

I would like to use this type in other places. We should eventually change firehose to use TypeScript. @alecmerdler has looked at that in the past.

Reply:
Put it in `utils/index.tsx` until we have `firehose.tsx`.

Reply:
ok, moved to `utils/index`